src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/RenameColumns.cs

```diff
@@ -265,5 +271,12 @@ public static ColumnReplacement Create(DName oldName, DName newName, DType type)
                 };
             }
         }
+
+        public override bool ComposeDependencyInfo(IRCallNode node, DependencyVisitor visitor, DependencyVisitor.DependencyContext context)
+        {
+            Utils2.FunctionSupportColumnNamesAsIdentifiersDependencyUtil(node, visitor);
```

Comment:
>Utils2 [](http://example.com/codeflow?start=12&length=6)

This is an extension method, so you can just say: 

node.FunctionSupportColumnNamesAsIdentifiersDependencyUtil(visitor);


Apply to other usages